Eatery/Models/CollegetownEatery.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  CollegetownEatery.swift
+//  Eatery
+//
+//  Created by William Ma on 4/8/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import CoreLocation
+import Foundation
+
+enum Rating: Double {
```

Comment:
Why is there a need for enumeration for this? I don't feel as if it's required or looks too clean

Reply:
I went too overboard with trying to push everything to be enums. I'll change it back and let backend know.